src/chunk.js

```diff
@@ -404,6 +407,13 @@ class Chunks {
    * @private
    */
   executeAsap_(idleDeadline) {
+    // If we've spent over 5 millseconds executing the
+    // last instruction yeild back to the main thread.
+    // 5 milliseconds is a magic number.
+    if (Date.now() - this.timeSinceLastExecution_ > 5) {
```

Comment:
Nit: Should `timeSinceLastExecution_` be called `lastExecutionTimestamp_`? as it stores the time stamp and not the time passed since the last execution